vspace/src/main/java/edu/asu/diging/vspace/core/services/IExhibitionAboutPageManager.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.diging.vspace.core.services;
+
+import java.util.List;
+
+import edu.asu.diging.vspace.core.model.impl.ExhibitionAboutPage;
+/**
+ * IExhibitionAboutPageManager allows to store and manage ExhibitionAboutPage.
+ * @author Avirup Biswas
+ *
+ */
+public interface IExhibitionAboutPageManager {
+    
+    /**
+     * This method fetches all the entries from table and
+     *  return a list of each entries from that table
```

Comment:
Should be:
"This method fetches all {@link ExhibitionAboutPage} entries and returns them."